File: Lesson2/app/src/main/java/ru/a777alko/lesson2/MainActivity.java
package ru.a777alko.lesson2;

import androidx.appcompat.app.AppCompatActivity;
import io.reactivex.disposables.Disposable;
import io.reactivex.functions.Consumer;

import android.os.Bundle;
import android.widget.EditText;
import android.widget.TextView;

import com.jakewharton.rxbinding3.widget.RxTextView;

public class MainActivity extends AppCompatActivity {

    private EditText editText;
    private TextView textView;
    private Disposable d;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        editText = findViewById(R.id.editText);
        textView = findViewById(R.id.textView);

        d = RxTextView.textChanges(editText)
                .subscribe(new Consumer<CharSequence>() {
                    @Override
                    public void accept(CharSequence charSequence) throws Exception {
                        textView.setText(charSequence);
                    }
                });
    }

    @Override
    protected void onStop() {
        super.onStop();
        if (d != null) {
            d.dispose();
        }
    }
}

File: Kotlin/app/src/main/java/ru/geekbrains/kotlin1/MainActivity.kt
package ru.geekbrains.kotlin1

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    var count : Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var btnTest = findViewById<Button>(R.id.btn_test)
        var txtView = findViewById<TextView>(R.id.txt_view)
        btnTest.setOnClickListener {
            txtView.text = "Кнопка нажата ${count++}"
        }
    }
}
